jupyterhub/jupyterhub/base/jupyterhub-dc.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     app: jupyterhub
   name: jupyterhub
 spec:
-  replicas: 1
```

Comment:
Assuming this is only for running JupyterHub in HA, the should not be included as the default for a JupyterHub deployment